isofit/core/fileio.py

```diff
@@ -354,26 +359,29 @@ class IO:
         isofit.root, "data", "earth_sun_distance.txt"
     )
 
-    def __init__(self, config: Config, forward: ForwardModel):
+    def __init__(self, config: Config, full_statevec: np.array):
```

Comment:
IO doesn't need the whole forward model object. It only needs 3 things from it: 
meas_wl, meas_fwhm, statevec_names